### DISCUSSION TITLE
Add "Download All" Button for Favorite Beatmaps

### DISCUSSION THREAD

**Discussion by Schweeeeeeeeeeeeeeee** [ppy/osu#23803] [Ideas] [answered]
Downloading all my beatmaps in Osu! can be a tedious task, especially when I frequently reset my PC. While this feature may primarily address my needs, I believe implementing a "Download All" button could greatly simplify the process and enhance the user experience for other Osu! players as well. This feature would be particularly valuable for users with large collections, saving them precious time and effort of individually clicking on every single beatmap. To ensure a streamlined interface, it would be ideal for the "Download All" button to be displayed exclusively on the user's own profile.

**Comment by Rekunan** [comment]
I don't think they'd want to add any mass downloading anywhere, especially in here, considering that, currently, you can have up to 1000 favorites.

**Comment by Schweeeeeeeeeeeeeeee** [comment]
A way to export all my maps from lazer would fix my issue.
Or a way to migrate from one lazer install to another.
May just be a specific issue though.

**Reply by peppy** [reply]
You can point lazer to any data folder and it will immediately pick up on it.

**Reply by Schweeeeeeeeeeeeeeee** [reply]
Thx didnt know, that fixes my problem.

**Reply by peppy** [reply, marked as answer]
In settings, you can choose "change folder location" and then select a data folder. lazer will ask if you want to use it before switching!
